internal/processing/helpers.go

```diff
@@ -54,6 +55,18 @@ func setAccessRuleKey(hasPathDuplicates bool, rule rulev1alpha1.Rule) string {
 	return rule.Spec.Match.URL
 }
 
+func setAuthorizationPolicyKey(hasPathDuplicates bool, ap *istiosecurityv1beta1.AuthorizationPolicy) string {
+	if hasPathDuplicates {
+		return fmt.Sprintf("%s:%s", ap.Spec.Rules[0].To[0].Operation.Paths, ap.Spec.Rules[0].To[0].Operation.Methods)
```

Comment:
perhaps is a good idea to check for nils when digging into Spec, making it more resilient